src/disco/containers/DiscoPane.js

```diff
@@ -96,11 +101,12 @@ export class DiscoPaneBase extends React.Component {
     // TODO: Add captions see https://github.com/mozilla/addons/issues/367
     /* eslint-disable jsx-a11y/media-has-caption */
 
-    const { AddonComponent, results, i18n } = this.props;
+    const { AddonComponent, errorHandler, results, i18n } = this.props;
     const { showVideo } = this.state;
 
     return (
       <div id="app-view" ref={(ref) => { this.container = ref; }}>
+        {errorHandler.hasError() ? errorHandler.renderError() : null}
```

Comment:
I notice we do this a lot, probably worth having `errorHandler.renderError()` just output `null` if no errors exist as it's often what we do... separate thing though.